systemd/systemd.go

```diff
@@ -692,12 +705,56 @@ func IsTimeout(err error) bool {
 	return isTimeout
 }
 
+func (l Log) parseLogRawMessageString(key string, sliceHandler func([]string) (string, error)) (string, error) {
+	valObject, ok := l[key]
+	if !ok || valObject == nil {
+		// NOTE: journalctl says that sometimes if a json string would be too
+		// large null is returned, so we may miss a message here
+		return "", fmt.Errorf("key %q missing from message", key)
+	}
+
+	// first try normal string
+	s := ""
+	err := json.Unmarshal(*valObject, &s)
+	if err == nil {
+		return s, nil
+	}
+
+	// try runes next, this is the case if journald thinks the output is not
+	// safe ascii
+	r := []rune{}
+	err = json.Unmarshal(*valObject, &r)
+	if err == nil {
+		return string(r), nil
+	}
+
+	// finally try list of strings
+	stringSlice := []string{}
```

Comment:
Is there any evidence that indicates we may need to use [][]rune too?

Reply:
I haven't seen evidence of it, but I also haven't dug into journald code deeply enough to know that it _can't_ happen, so sure let's just be extra safe and handle that here too :smile:

Reply:
added and tested now